### ISSUE TITLE
CRASHed randomly

### ISSUE THREAD

**Issue by inshiro** [Adamantcheese/Kuroba#273]
Just scrolling through and reading threads when it suddenly crashed.
```ruby
E/Kuroba | UNCAUGHT: java.lang.NullPointerException: Attempt to read from field 'com.github.adamantcheese.chan.core.model.orm.Loadable com.github.adamantcheese.chan.core.model.ChanThread.loadable' on a null object reference
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.ThreadLayout.showPosts(ThreadLayout.java:269)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.showPosts(ThreadPresenter.java:1117)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.onListScrolledToBottom(ThreadPresenter.java:623)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.ThreadListLayout.lambda$onRecyclerViewScrolled$0$ThreadListLayout(ThreadListLayout.java:164)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.-$$Lambda$ThreadListLayout$wCwyHJXO34_chrEhTvPk0rZTQGs.run(Unknown Source:2)
E/Kuroba | UNCAUGHT: 	at android.os.Handler.handleCallback(Handler.java:873)
E/Kuroba | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/Kuroba | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:193)
E/Kuroba | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:6718)
E/Kuroba | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/Kuroba | UNCAUGHT: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/Kuroba | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
E/Kuroba | UNCAUGHT: .

```

**Comment by Adamantcheese** [comment]
This is probably one of those crashes that will be fixed with #247.

**Comment by K1rakishou** [comment]
Yeah, it's probably the multithreading issue again which is fixed in #247
